debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java

```diff
@@ -969,6 +969,7 @@ protected void parseRenameTable(Marker start) {
         tokens.consume("TO");
         TableId to = parseQualifiedTableName(start);
         databaseTables.renameTable(from, to);
+        signalAlterTable(from, to, start);
```

Comment:
Unfortunately, this will capture only the portion of the `RENAME TABLE` statement up till the end of the new name of the table. For example, given:

    RENAME TABLE a TO b, c TO d, e TO f

The first emitted DDL statement will be

    RENAME TABLE a TO b

the second emitted DDL statement will be 

    RENAME TABLE a TO b, c TO d

while the third emitted DDL statement will be

    RENAME TABLE a TO b, c TO d, e TO f`

This needs to be tweaked in order to work properly. I'll merge this and adjust.